boards/FSM/Src/App/App_SetPeriodicCanSignals.c

```diff
@@ -17,3 +18,34 @@ void App_SetPeriodicCanSignals_SecondaryFlowMeter(
     App_CanTx_SetPeriodicSignal_SECONDARY_FLOW_RATE(
         can_tx, App_FlowMeter_GetFlowRate(flow_meter));
 }
+
+void App_SetPeriodicCanSignals_LeftWheelSpeedSensor(
```

Comment:
`*const` for arguments in `.c`

Reply:
I got rid of this function

Reply:
After https://github.com/UBCFormulaElectric/Consolidated-Firmware/pull/582#discussion_r417063611 I should also probably get rid of the SetPeriodicSignals for the flow meters as well. Should I do it in this PR?

Reply:
Sure I don't mind